.history/game_20230912214758.js

```diff
@@ -0,0 +1,71 @@
+const prompt = require {"prompt-sync"} ();
+
+function getComputerChoice() {
+    let choices = ["Rock", "Paper", "Scissors"];
+    let choice = Math.floor(Math.random() * choices.length);
+    console,log("Computer picked" + " " + choices[choice]);
+    return choices[choice];
+}
+
+function userChoice(){
+    let userInput = prompt(
+        "Enter your choice: "
+        ).toLowerCase();
+
+        if (userInput == "rock") {
+            console.log("You Picked Rock");
+            return "rock";
+        }else if (userInput == "paper") {
+            console.log("You Picked Paper");
+            return "paper"
+        }else if (userInput == "scissors") {
+            console.log("You Picked Scissors");
+            return "scissors"
+        } else {
+            console.log("Invalid Input");
+        }            
+    }
+
+    function game() {
+        let playerSelection = userChoice();
+        let computerSelection = getComputerChoice();
+
+        if (playerSelection == computerSelection) {
+            console.log("Its a Tie try Again");
+        } else if (playerSelection === "rock") {
+            if (computerSelection === "scissors") {
+                console.log("You Win");
+                return "win"
+            } else if (computerSelection === "paper") {
+                console.log("You Lose");
+                return "lose"
+            } else if (computerSelection === "rock") {
+                console.log("Its a Tie");
+                return "tie"
+            }
+        } else if (playerSelection === "paper") {
+            if (computerSelection === "rock") {
+                console.log("You Win");
```

Comment:
Pay more attention to details, the respond was supposed to be "You won, paper covers rock" and not just you won